keel-web/src/test/kotlin/com/netflix/spinnaker/keel/apidocs/ApiDocTests.kt

```diff
@@ -191,37 +193,23 @@ class ApiDocTests : JUnit5Minutests {
         }
       }
 
-    test("schema for a sealed class is oneOf the sub-types") {
-      at("/\$defs/ImageProvider")
-        .isObject()
-        .has("oneOf")
-        .path("oneOf")
-        .isArray()
-        .findValuesAsText("\$ref")
-        .containsExactlyInAnyOrder(
-          "#/\$defs/ArtifactImageProvider",
-          "#/\$defs/JenkinsImageProvider",
-          "#/\$defs/ReferenceArtifactImageProvider"
-        )
-    }
-
-    imageProviderTypes.map(Class<*>::getSimpleName)
+    securityGroupRuleTypes.map(Class<*>::getSimpleName)
       .forEach { type ->
-        test("ImageProvider sub-type $type has its own schema") {
+        test("SecurityGroupRule sub-type $type has its own schema") {
           at("/\$defs/$type")
             .isObject()
         }
       }
 
-    test("schema for ImageProvider is oneOf the sub-types") {
-      at("/\$defs/ImageProvider")
+    test("schema for SecurityGroupRule is oneOf the sub-types") {
+      at("/\$defs/SecurityGroupRule")
         .isObject()
         .has("oneOf")
         .path("oneOf")
         .isArray()
         .findValuesAsText("\$ref")
         .containsExactlyInAnyOrder(
-          imageProviderTypes.map { "#/\$defs/${it.simpleName}" }
+          securityGroupRuleTypes.map { "#/\$defs/${it.simpleName}" }
         )
     }
```

Comment:
The point of these tests was to ensure the JSON schema renders sealed class hierarchies correctly, so I just pointed it at a different one since `ImageProvider` is no longer a hierarchy at all